knowledge-base/common-change-selected-localization-keys.md

```diff
@@ -0,0 +1,197 @@
+---
+title: Localize selected localization keys with custom strings
+description: How to modify selected localized strings without maintaining the complete version of the localization resource files
+type: how-to
+page_title: Localize selected localization keys with custom strings
+slug: common-kb-localize-selected-localization-keys
+position:
+tags: telerik, blazor, localization, key, resources, files, partial, string
+ticketid: 1601981, 1607465, 1604787
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>Telerik UI for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
```

Comment:
Are these four questions taken from four different tickets? They sound unrelated and don't build coherent text or description. This approach doesn't give significant value to the article as it represents a kind of shotgun approach where you throw all possible questions and keywords and hope that something sticks.
A much better approach is to tie them logically together and build a description that represents a single use case with multiple requirements.